index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Kareem Wilson</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <ul class="nav-list">
+        <li class="nav-item"><a href="#Welcome">Portofolio</a></li>
+        <li class="nav-item"><a href="#About">About</a></li>
+        <li class="nav-item"><a href="#Contact">Contact</a></li>
```

Comment:
Kindly include a header tag which will be a container for the logo and the navigation list. Doing so makes for better semantics.